### PR TITLE
News Update

### PR DESCRIPTION
**Pull request by TheEnthusiasticAs**

### CODE REVIEW

**Review by gitr0n1n — CHANGES_REQUESTED** [review]
@TheEnthusiasticAs Duplicate Coinbase article. Please fix.

**Review by gitr0n1n — APPROVED** [review]
LGTM

**Review by gitr0n1n — APPROVED** [review]
LGTM

File: content/news/media/media.collection.en.yaml [hunk 1]
```diff
@@ -1,3 +1,38 @@
+-
+  date: 2020-09-10
+  link: https://storj.io/blog/2020/09/using-tardigrade-and-the-decentralized-cloud-to-speed-up-sync-times-for-geth/?utm_source=etc-labs&utm_medium=twitter
```
Comment (mikeyb) [inline content/news/media/media.collection.en.yaml:3]: Can we remove the `?utm_source=etc-labs&utm_medium=twitter` part of the URL on this?

**Review by mikeyb — CHANGES_REQUESTED** [review]
Looks good minus the 1 url

File: content/news/media/media.collection.en.yaml (hunk 1 above)
Comment (TheEnthusiasticAs) [reply content/news/media/media.collection.en.yaml:3]: yes

**Review by mikeyb — APPROVED** [review]
LGTM

**Pull request merged**